generators/server/files.js

```diff
@@ -357,18 +357,6 @@ const serverFiles = {
         },
     ],
     serverJavaAuthConfig: [
-        {
-            condition: generator =>
-                !generator.reactive &&
-                (generator.databaseType === 'sql' || generator.databaseType === 'mongodb' || generator.databaseType === 'couchbase'),
-            path: SERVER_MAIN_SRC_DIR,
-            templates: [
-                {
-                    file: 'package/security/SpringSecurityAuditorAware.java',
-                    renameTo: generator => `${generator.javaDir}security/SpringSecurityAuditorAware.java`,
```

Comment:
Don't forget to remove old files in `cleanup.js`

Reply:
good catch. Thanks you